examples/benchmarks/text_classification_imdb.ipynb

```diff
@@ -1,1301 +1,1325 @@
 {
```

Comment:
<p>Delete this output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Toloka/toloka-kit/pull/83/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>